caramel_launcher.ini

```diff
@@ -0,0 +1,80 @@
+###
+# app configuration
+# http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/environment.html
+###
+[app:main]
+use = egg:caramel
+
+# Point the following two settings on where you want your CA certificate & Key to live
+ca.cert = %(ca_cert)s
+ca.key = %(ca_key)s
+
+# This causes all certs to be backdated to the age of the start cert.
+# This is an ugly workaround for our embedded systems that lack RTC.
+backdate = False
+# Default to 48 hour certs
+lifetime.short = %(life_short)s
```

Comment:
Is this a good idea in that it forces the configuration to be done somewhere else, rather than having a default value?

Reply:
From my understanding pserve doesn't allow for defaults when using variable assignment, so if you want variable assignment this is the simplest workaround.